amethyst_assets/src/loader.rs

```diff
@@ -163,10 +163,11 @@ impl Loader {
     fn context<C>(&self) -> &Arc<C>
         where C: Context + 'static,
     {
-        let context = self.contexts
+        let context: &Box<Any + Send + Sync> = self.contexts
             .get(&TypeId::of::<C::Asset>())
             .expect("Assets need to be registered with `Loader::register`.");
 
+        let context: &Any = context; // Won't compile without this one
```

Comment:
No.
`Any + Send + Sync` doesn't has `downcast_ref` method
`Any` has it, `Any + Send` has it, even `Any + Send + 'static` has it. But not `Any + Send + Sync`
So I have to manually cast it to `&Any`
It can be `(context as &Any).downcast_ref()` or as I done here.